components/sidebar/BUILD.gn

```diff
@@ -6,6 +6,8 @@
 import("//brave/components/sidebar/buildflags/buildflags.gni")
 
 static_library("sidebar") {
+  assert(enable_sidebar)
```

Comment:
also `enable_sidebar = is_win || is_mac || is_linux` -> `enable_sidebar = toolkit_views` in buildflags.gni